Kubernetes/legos/k8s_check_service_pvc_utilization/k8s_check_service_pvc_utilization.py

```diff
@@ -35,7 +36,7 @@ def k8s_check_service_pvc_utilization_printer(output):
             print(f"PVC: {pvc['pvc_name']} - Utilized: {pvc['used']} of {pvc['capacity']}")
         print("-" * 40)
 
-
+@beartype
```

Comment:
this is not needed in the committed code. this gets added by the code snippet script